Lib/test/test_typing.py

```diff
@@ -603,22 +603,16 @@ class C(Generic[T]): pass
             ('generic[T]',                        '[int]',                   'generic[int]'),
             ('generic[T]',                        '[int, str]',              'TypeError'),
             ('generic[T]',                        '[tuple_type[int, ...]]',  'generic[tuple_type[int, ...]]'),
+            ('generic[T]',                        '[*tuple_type[int]]',      'generic[int]'),
             # Should raise TypeError: a) according to the tentative spec,
```

Comment:
We can get rid of the comment here and a few lines below now that these cases do indeed raise a TypeError.